Microsoft.Research/ManagedContract.Setup/DotNet.Contracts.targets

```diff
@@ -0,0 +1,14 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <CodeContractsInstallDir>$(MSBuildThisFileDirectory)</CodeContractsInstallDir>
```

Comment:
does it set it unconditionally? A condition could be added in order to be able to override the value in the project file.